### DISCUSSION TITLE
New Mod Idea: Object Spacing.

### DISCUSSION THREAD

**Discussion by unrealic** [ppy/osu#17128] [Ideas]
**New Mod Idea: Object Spacing.**

Can change the distance between objects (like the difficulty adjust mod, could be implemented into it) for more difficulty without affecting score. Harder jumps + spaced streams would be cool to practise with a mod. It could also decrease object distance if you are struggling to hit a pattern and need a build-up to it.

I understand this can be difficult as increasing some objects' distance may lead to them coming off of the screen, and to ensure this doesn't happen the angle from the previous object would have to change, but I think it would still be cool and useful for the community.

**Comment by peppy** [comment]
random mod already has the ability to do this to a certain extent. could share implementation.

**Reply by hlysine** [reply]
Sounds like another fun idea to work on :P I'll start by extracting random mod's logic.

Update: PR created #17144
